### PR TITLE
Trying to fix `aboutanime`

### LINKED ISSUE #687: Command 'aboutanime' raised an exception: ValueError: No such id on MyAnimeList
**Describe the bug**
Hybrid command raised an error: Command 'aboutanime' raised an exception: ValueError: No such id on MyAnimeList

**To Reproduce**
1. Type any random mal_id in `aboutanime` comand
2. Wait for some time
3. See error

**Expected behavior**
It should not error and quietly say that dosen't exists in MAL database.

**Additional context**
https://discordapp.com/channels/920190307595874304/920190310942908513/988830767763292172

https://discord.gg/vfXHwS3nmQ

### PR DESCRIPTION
**Pull request by eitozx**
Signed-off-by: Eito <iameitozx@gmail.com>

## Summary
This is suppose to fix #687 . 
Honestly saying, I think this should fix, has this been tested? sorry i couldn't, I asked u for the cog file but it turns out that it is importing various stuff from other files as well. If you can merge this, fine, or otherwise I suggest you to make a guide for contributors please lmao 💀 

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Chii-Onan** [comment]
Hi, I'd just like to address a couple of things.

> sorry i couldn't, I asked u for the cog file but it turns out that it is importing various stuff from other files as well. If you can merge this, fine, or otherwise I suggest you to make a guide for contributors please lmao 💀

I don't really think it would've hard to find a command with crtl + f but kk.

Next, this PR doesn't do much... In the try and except, we won't always error because the anime wasn't found. There can be a multitude of reasons and as such this message is very misleading, if that makes sense.

**Comment by eitozx** [comment]
> Next, this PR doesn't do much... In the try and except, we won't always error because the anime wasn't found. There can be a multitude of reasons and as such this message is very misleading, if that makes sense.

I find it hard to start the bot without the guide to set up the bot. That's the simple reason cuz there are many changes in set-up since last year. And i agree with your point in terms of error. Either way, I don't mind making changes whereever required if i get the set-up guide

**Comment by Dhruvacube** [comment]
> > Next, this PR doesn't do much... In the try and except, we won't always error because the anime wasn't found. There can be a multitude of reasons and as such this message is very misleading, if that makes sense.
> 
> I find it hard to start the bot without the guide to set up the bot. That's the simple reason cuz there are many changes in set-up since last year. And i agree with your point in terms of error. Either way, I don't mind making changes whereever required if i get the set-up guide

yeah going to set up that only!

File: minato_namikaze/cogs/anime_and_vocaloids/anime_and_waifu.py [hunk 1]
```diff
@@ -183,7 +183,11 @@ async def animesearch(self, ctx: "Context", *, anime_name: str):
     async def aboutanime(self, ctx: "Context", mal_id: int):
         """Displays about the anime using the MAL ANIME ID. get it by using animesearch command."""
         await ctx.send(":mag: Searching...", delete_after=5)
-        anime = Anime(int(mal_id))
+        try:
+            anime = Anime(int(mal_id))
+        except:
```
Comment (Dhruvacube) [inline minato_namikaze/cogs/anime_and_vocaloids/anime_and_waifu.py:188]: please here do change this to this

Comment (Dhruvacube) [reply minato_namikaze/cogs/anime_and_vocaloids/anime_and_waifu.py:188]:
```python
except ValueError:
```

**Review by Dhruvacube — CHANGES_REQUESTED** [review]

**Review by Dhruvacube — APPROVED** [review]

**Pull request merged**